Description: Disable failing asserts
 iso-codes has long moved on beyond that which pycountry knows.
 This means the count of countries, subdivisions, etc are out of
 date.  Rather than risk an FTBFS when iso-codes moves on once more
 simply comment out the specific asserts, leaving the rest of the
 testing intact.
Author: Daniel Silverstone <dsilvers@digital-scurf.org>
Origin: other
Bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=821892
Forwarded: no
Reviewed-by: Jonathan Wiltshire <jmw@debian.org>
Last-Update: 2017-02-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/pycountry/tests/test_general.py
+++ b/src/pycountry/tests/test_general.py
@@ -76,7 +76,7 @@
 
 
 def test_subdivisions_directly_accessible():
-    assert len(pycountry.subdivisions) == 5123
+    #assert len(pycountry.subdivisions) == 5123
     assert isinstance(list(pycountry.subdivisions)[0], pycountry.db.Data)
 
     de_st = pycountry.subdivisions.get(code="DE-ST")
@@ -104,7 +104,7 @@
 
 
 def test_scripts():
-    assert len(pycountry.scripts) == 182
+    #assert len(pycountry.scripts) == 182
     assert isinstance(list(pycountry.scripts)[0], pycountry.db.Data)
 
     latin = pycountry.scripts.get(name="Latin")
@@ -114,7 +114,7 @@
 
 
 def test_currencies():
-    assert len(pycountry.currencies) == 170
+    #assert len(pycountry.currencies) == 170
     assert isinstance(list(pycountry.currencies)[0], pycountry.db.Data)
 
     argentine_peso = pycountry.currencies.get(alpha_3="ARS")
@@ -124,7 +124,7 @@
 
 
 def test_languages():
-    assert len(pycountry.languages) == 7847
+    #assert len(pycountry.languages) == 7847
     assert isinstance(list(pycountry.languages)[0], pycountry.db.Data)
 
     aragonese = pycountry.languages.get(alpha_2="an")
